### ISSUE TITLE
Crash on lan server just sitting there

### ISSUE THREAD

**Issue by shugnk** [Amaxter/Core-Zero#21]
The client crashes when on a lan server, after a little while. Im not doing anything either when it crashes. Like I could just be sitting still 
http://pastebin.com/h9aYDr1R

**Comment by Amaxter** [comment]
I'll look into it, not much I can do with a crash like that as it's not very clear what mod it's even from. This only happens in LAN?

**Comment by shugnk** [comment]
yes, and randomly at that. We were not even doing anything. The game was just idling.

**Comment by Amaxter** [comment]
I find it hard to believe it happens in just LAN, try playing in SP and see if it happens also, I bet it will.

**Comment by shugnk** [comment]
Well, this is happening to a friend whom didnt think it was worth posting a issue about. It happens over the lan server. I am the host whilest he is the player on the server. He doesnt do anything and just crashes. It usually happens between 2-8 minutes. This server is over hamachi. The first time he crashed was because he teleported to me with the map addon. The following times where because he loaded into the game. I did delete his player data completely and in less than 5 minutes, he crashed once more. Im not sure what exactly it is. From the crash log it seems that maybe a packet is fragmented or incomplete, is being sent to him. Or from looking up the error, ive been able to find some almost identical error logs from other mod packs; with the error being the server wanted to spawn animals even though it knew it couldn't.

**Comment by Amaxter** [comment]
Perhaps it's an oddity with his connection. I'm not sure how well some mods work over Hamachi as packet transfer is greatly increased.

**Comment by shugnk** [comment]
hmm, well, we havent had this issue before. And on another note. The quests on my client got messed up or something. When I finished all the primary quests in the first chapter, I couldnt start the next chapter. I didnt have the option too or anything like that. I even tried to edit the quests to make them playable but to no avail. So the trigger to the second chapter wasnt working or something along those lines. I tried also to reset all my quests and go back with the OP quest book at get back to the same quest it stopped me at. It also glitched up and I could only get to half of the first chapter. It wouldnt allow the other quests to show up.
